src/game/models/mod.rs

```diff
@@ -2,156 +2,16 @@
 
 use std::f64;
```

Comment:
Oh... wow. It's because you are actually importing the type `f64`??

```rust
use std::f64; // No warning.

fn main() {
    let x: f64 = 0.0;
    println!("{}", x);
}
```